sdcm/scan_operation_thread.py

```diff
@@ -46,6 +46,11 @@ def __init__(self, db_cluster: [BaseScyllaCluster, BaseCluster], duration: int,
         self.termination_event = termination_event
         self.log = logging.getLogger(self.__class__.__name__)
         self._thread = threading.Thread(daemon=True, name=self.__class__.__name__, target=self.run)
+        self.username, self.password = self.get_credentials()
```

Comment:
instead of using this way, it should be a property, as the credentials provided by `get_db_aut()` may change during the test...

Reply:
Funny, it looks like these parameters are totally ignored and not in use according to this:
https://github.com/scylladb/scylla-cluster-tests/blob/b26cd343e2710e4b3e8956ac0b07abaf72e4d5cf/sdcm/cluster.py#L3264

Pehaps i'll try removing this code and see if it works without it.

Reply:
every time a session is created, we need to use the updated credentials...
but once the session is created, it doesn't make any difference, as it is used only once for each session

Reply:
test passed ok after removing this code in:
https://jenkins.scylladb.com/job/scylla-staging/job/Longevity_yaron/job/longevity-large-partition-4days-test-repro/22/

i'll go ahead and remove this code from this PR then. I can send a separate PR to remove the `user` and `password` parameters from the relevant methods' definition.

Reply:
fixed this PR.

Reply:
opened https://github.com/scylladb/scylla-cluster-tests/pull/4906

Reply:
it is in use... as i said before, it is called when creating a session, and if there are username and password configured (one example is LDAP tests)